docs/en-US/Update-PatServerToken.md

```
@@ -0,0 +1,212 @@
+online version:
+schema: 2.0.0
+---
+
+# Update-PatServerToken
+
+## SYNOPSIS
+Refreshes the authentication token for a stored Plex server.
+
+## SYNTAX
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)